requirements.txt

```diff
@@ -0,0 +1,91 @@
+alembic==1.5.3
```

Comment:
It seems you are not using all these packages. Can you remove unwanted packages?

Reply:
Not sure! What I'm using in the packages in the local repository. 
Definitely, I remove the unnecessary packages in the local repository

Reply:
You did not understand how using packages works. Read up on it. Try to create the virtual environment first before start working on anything.

Reply:
Definitely, I will create a virtual environment.